docs/general/attacks.md

```diff
@@ -0,0 +1,81 @@
+---
+id: attacks
+title: Protection from Attacks
+sidebar_label: Attack Protection
+description: How to Prevent from being a Victim of an Attack
+keywords: [attack, protection]
+slug: ../attacks
```

Comment:
Same here. An attacks page on Polkadot Wiki could also mean attacks on the protocol, which this page does not cover. Rename this to reflect the narrow scope of transaction based attacks.

Reply:
Renamed the slug to reflect this feedback